### ISSUE TITLE
🆕 Software Suggestion |  Hideout

### ISSUE THREAD

**Issue by techmindful** [privacytools/privacytools.io#2354]
## Basic Information

**Name:** Hideout
**Category:** Self-hosted chat service
**URL:** https://github.com/techmindful/hideout

## Description
I'm the developer of Hideout. The GitHub repo linked above has a very detailed introduction in the README. But in short, I realized it's still too much work for some people to install an app like Signal and register an account. If they do sign up, most of the time I'll also be the only contact they'll ever have on that app. So I made Hideout. It's built on the idea of disposable chat. Except that it offers a stronger guarantee of privacy, without needing the chat room to expire! After a persistent chat room is created, all the users can bookmark the room page, and come back to continue the conversation, without having to create new chat rooms. Yet the chat room remains private.

Hideout also tries to be as feature-rich as possible, so when people choose privacy, they aren't opting for a lackluster alternative.

Hideout is built with functional programming. More reliable!

Hideout does need to be self-hosted.

## Why I am making the suggestion
Described above.

## My connection with the software
I'm the author.

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by ph00lt0** [comment]
TLDR: Feels a bit like a POC, not something average user would appreciate. Although it's nice project you made, I don't think this should be listed.

There are many user friendly alternatives like RocketChat, Element, Session, I do not see the advantage of this tool.
This web app doesn't come with the features most users would be looking for like video, document sharing, end-to-end encryption and above all a user-friendly UI.

The goal of PTIO is to make privacy accessible to masses, not to list every project that does something in the mind of privacy so don't feel bad about my comment.

**Comment by techmindful** [comment]
@ph00lt0 Thanks for the feedback. It does look like a proof-of-concept at this point. It will have a lot more selling point after voice/video chat and file sharing are implemented. By then, Hideout will have the usual features other apps have, *plus* removing the need for any account creation and software installation.

**Comment by techmindful** [comment]
Following the comment above, I'm not sure if I should just keep this issue open, or close it and re-open after more features are implemented? Maybe an admin of privacytools.io can let me know?
